schema.sql

```diff
@@ -32,3 +32,27 @@ ADD COLUMN species_id  INT references species(id);
 
```

Comment:
- [ ] Kindly add mandatory information in your readme file because it is looking like a template. Kindly insert your data in the readme file such as Project Description, Key Feature, Future Feature, Getting Started, and License Link :+1: